CodeMaid.UnitTests/Formatting/HeaderFormattingTests.cs

```diff
@@ -22,12 +22,6 @@ namespace SteveCadwallader.CodeMaid.UnitTests.Formatting
     [TestClass]
     public class HeaderFormattingTests
     {
-        [TestInitialize]
```

Comment:
I think by convention TestInitialize methods are usually at the top of a unit test file.  Any particular reason to move it to the bottom?  I'm guessing maybe our reorganization did it automatically and that should be a backlog card to make an exception for some unit test methods.